source/tutorials/longitudinal.rst

```diff
@@ -275,7 +275,7 @@ Let's test this out on the ECAM dataset. First download a table to work with:
      --p-state-column month \
      --p-individual-id-column studyid \
      --p-n-estimators 10 \
-     --p-random-state 123 \
+     --p-random-state 17 \
```

Comment:
Just curious - why the change away from `123` here, when we are setting `random-state` to 123 for all sample-classifier methods? And is there a reason we've set `maturity-index`'s `random-state` differently from the other passed values (i.e. `1010101`)?